app/views/users/profile.html.erb

```diff
@@ -125,7 +125,7 @@
         <hr />
         <div id="follow_form" class="col-md-offset-4">
           <% if current_user.following?(@profile_user) %>
-            <a id="profile-unfollow-button" class="btn btn-primary" data-method="delete" data-remote="true" href="../relationships/<%= @profile_user.id %>"><%= translation('users.profile.unfollow') %></a>
+            <a id="profile-unfollow-button" class="btn btn-light" data-method="delete" data-remote="true" href="../relationships/<%= @profile_user.id %>"><%= translation('users.profile.unfollow') %></a>
```

Comment:
I'm really sorry for the incovenience @garimavs, you are right about the button color. Can you switch it back to `btn-secondary`. Thanks!

Reply:
No worries! I'll do it again